src/components/MainPage/Post/Post.jsx

```diff
@@ -67,18 +171,51 @@ function Post({
 					setLike={setLike}
 					saveClicked={saveClicked}
 					handleSaveClicked={handleSaveClicked}
+					handleCommentClicked={handleCommentClicked}
 				/>
-				<div className="comment-section">
-					<ProfilePicture
-						imgSrc={imgSrcCommenter}
-						height="35px"
-					/>
-					<input
-						type="text"
-						placeholder="What do you think?"
-						className="comment-input"
-					/>
-				</div>
+
+				{showComments ? (
+					<div className="comments">
+						<div className="comment-section">
+							<ProfilePicture
+								imgSrc={imgSrcCommenter}
+								height="35px"
+							/>
+							<div className="input-container">
+								<input
+									type="text"
+									placeholder="What do you think?"
+									className="comment-input"
+									value={comment}
+									onKeyDown={handleKeyDown}
+									onChange={(e) => setComment(e.target.value)}
+								/>
+								<button
+									className="submit-comment"
+									onClick={handleCommentSubmit}>
```

Comment:
handleCommentSubmit should have logic to not allow empty string to be passed when commenting. i tried entering empty comment and it crashes users session. i need to log out and in to proceed doing anything

Reply:
Good point!